openreview/api/client.py

```diff
@@ -271,7 +271,7 @@ def get_edge(self, id):
         return: Edge object with its information
         :rtype: Edge
         """
-        response = requests.get(self.tags_url, params = {'id': id}, headers = self.headers)
+        response = requests.get(self.edges_url, params = {'id': id, 'trash': 'true' if trash == True else 'false'}, headers=self.headers)
```

Comment:
I don't think you need to translate the value.

Reply:
do you mean params = {'id': id, 'trash': trash} ?
this will fail api validation that trash must be a boolean value

Reply:
No need of translation, the API should receive a boolean, take a look at this one:

https://github.com/openreview/openreview-py/blob/master/openreview/openreview.py#L1012